go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/hashicorp/go-multierror v1.1.1 // indirect
 	github.com/hashicorp/go-retryablehttp v0.7.5 // indirect
 	github.com/hashicorp/go-slug v0.15.0
-	github.com/hashicorp/go-tfe v1.51.0
+	github.com/hashicorp/go-tfe v1.52.1-0.20240501135417-86d24e41cec9
```

Comment:
Just to make sure we are on the same page: I think we will want to wait until 1.52.1 is released, and use that tag instead of the specific commit (as seen here)
This should work for testing and validation in the mean time though!

Reply:
Yes, we are on the same page! I updated this to point specifically to my branch so that @emlanctot could easily pull the changes pointed at my `go-tfe` changes without having to do it manually